### PR TITLE
Feature/frontend screen upload default parameter settings

### PR DESCRIPTION
**Pull request by Mazar-Shaik**

### CODE REVIEW

**Review by rekhadhankhar2018 — APPROVED** [review]

**Review by rekhadhankhar2018 — APPROVED** [review]

File: src/EPR.Calculator.Frontend.UnitTests/UnitTest1.cs
[the remark this answers is not in the document]
Comment (Mazar-Shaik) [reply src/EPR.Calculator.Frontend.UnitTests/UnitTest1.cs:1]: done

[the remark this answers is not in the document]
Comment (rekhadhankhar2018) [reply src/EPR.Calculator.Frontend.UnitTests/UnitTest1.cs:1]: I have already removed that in Main

[the remark this answers is not in the document]
Comment (udaydefra) [reply src/EPR.Calculator.Frontend.UnitTests/UnitTest1.cs:1]: Thanks

**Review by udaydefra — APPROVED** [review]
@Mazar-Shaik  LGTM

**Pull request merged**